### PR TITLE
cctz: add version 2.4

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **cctz/2.4**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
I sent a PR to the upstream, trying to update the current runtime folder destination: https://github.com/google/cctz/pull/285

File: recipes/cctz/all/conanfile.py
[the remark this answers is not in the document]
Comment (toge) [reply recipes/cctz/all/conanfile.py:1]: @uilianries 
Thank you for your review!
Fixed.

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by uilianries** [comment]
> I sent a PR to the upstream, trying to update the current runtime folder destination: [google/cctz#285](https://github.com/google/cctz/pull/285)

Good news, the upstream accepted the PR. On next release we will not need those patches anymore!

**Pull request merged**